test/unit_test/runtime/magnitude_test.cpp

```diff
@@ -218,8 +287,50 @@ TEST_CASE("Can raise Magnitudes to rational powers")
   }
 }
 
+TEST_CASE("can distinguish integral, rational, and irrational magnitudes")
+{
+  SECTION("Integer magnitudes are integral and rational")
+  {
+    auto check_rational_and_integral = [](Magnitude auto m) {
+      CHECK(is_integral(m));
+      CHECK(is_rational(m));
+    };
+    check_rational_and_integral(as_magnitude<1>());
+    check_rational_and_integral(as_magnitude<3>());
+    check_rational_and_integral(as_magnitude<8>());
+    check_rational_and_integral(as_magnitude<412>());
```

Comment:
```suggestion
    check_rational_and_integral(as_magnitude<412>());
    check_rational_and_integral(as_magnitude<ratio{1, 1}>());
```
This should pass, right?

Reply:
Yep, it does.  Added.

FWIW, both this and the other added test turn out to be identical to `as_magnitude<1>()`.